packages/contract-call-decoder/tsconfig.json

```
@@ -0,0 +1,47 @@
+{
+  "compilerOptions": {
+    "incremental": true,
+    "target": "es6",
+    "outDir": "build/main",
+    "rootDir": "src",
+    "moduleResolution": "node",
+    "module": "commonjs",
+    "declaration": true,
+    "inlineSourceMap": true,
+    "esModuleInterop": true /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */,
+    "resolveJsonModule": true /* Include modules imported with .json extension. */,
+
+    // "strict": true /* Enable all strict type-checking options. */,
```

Let's turn strict on
```suggestion
    "strict": true /* Enable all strict type-checking options. */,
```